schema.sql

```diff
@@ -1,5 +1,12 @@
 /* Database schema to keep the structure of entire database. */
+postgres=# CREATE DATABASE vet_clinic;
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT GENERATED ALWAYS AS IDENTITY,
```

Comment:
- [ ] <b>OPTIONAL:</b> It would be nice to add `NOT NULL` constraint on `ID` just to make sure that it can't accept null values.